docs/requirements.txt

```
@@ -1,4 +1,5 @@
-Sphinx==3.3.0
-sphinx-material==0.0.32
-sphinx-tabs==1.3.0
-sphinx-jinja==1.1.1
+sphinx
+docutils==0.17
+git+http://github.com/SymbiFlow/sphinx_materialdesign_theme.git@master#egg=sphinx_symbiflow_theme
```

This is the wrong theme. 
```suggestion
git+https://github.com/SymbiFlow/sphinx_symbiflow_theme.git#egg=sphinx-symbiflow-theme
```